corehq/apps/cloudcare/static/cloudcare/js/formplayer/app.js

```diff
@@ -144,8 +144,12 @@ FormplayerFrontend.on("start", function (options) {
         Backbone.history.start();
         // will be the same for every domain. TODO: get domain/username/pass from django
         if (this.getCurrentRoute() === "") {
-            if (options.previewMode) {
-                FormplayerFrontend.trigger("app:preview", options.apps[0]['_id']);
+            if (options.phoneMode) {
```

Comment:
trying to be smart with naming. phoneMode should get you to view the app in the single app mode. preview mode should work in both phone and regular mode